components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift

```diff
@@ -1,12 +1,12 @@
 /*
  Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
-
+ 
  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
-
+ 
```

Comment:
Nit: Please remove the added white space on the empty lines throughout.  It will help make the actual changes more obvious as well as keeping the formatting cleaner.